src/wp-includes/rest-api/endpoints/class-wp-rest-font-collections-controller.php

```diff
@@ -0,0 +1,275 @@
+<?php
+/**
+ * Rest Font Collection Controller.
+ *
+ * This file contains the class for the REST API Font Collection Controller.
+ *
+ * @package    WordPress
+ * @subpackage Font Collection
+ * @since      6.4.0
+ */
+
+/**
+ * Font Collection Controller class.
+ *
+ * @since 6.4.0
+ */
+class WP_REST_Font_Collections_Controller extends WP_REST_Controller {
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 6.4.0
+	 */
+	public function __construct() {
+		$this->rest_base = 'font-collections';
+		$this->namespace = 'wp/v2';
+	}
+
+	/**
+	 * Registers the routes for the objects of the controller.
+	 *
+	 * @since 6.4.0
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_font_collections' ),
+					'permission_callback' => array( $this, 'update_font_library_permissions_check' ),
+				),
+				'schema' => array( $this, 'get_font_collections_schema' ),
+			)
+		);
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/(?P<id>[\/\w-]+)',
+			array(
+				'args'   => array(
+					'id' => array(
+						'description' => __( 'The id of a font collection.' ),
```

Comment:
```suggestion
						'description' => __( 'Unique identifier for the post.' ),
```

Matches the posts controller. Alternatively, capitalize ID

Reply:
Noting that the new font family and font face controllers re-use the `id` description from the post endpoint schema: https://github.com/WordPress/gutenberg/blob/4a16dd2a3db956115797a86daf2dc2a7b2665ba3/lib/compat/wordpress-6.5/fonts/class-wp-rest-font-families-controller.php#L285